app/helpers/login.ts

```diff
@@ -0,0 +1,19 @@
+import { Keypair } from "stellar-sdk";
+
+function isSecretKeyValid(secretKey: string): boolean {
+  try {
+    Keypair.fromSecret(secretKey);
+    return true;
```

Comment:
This is not necessary it will be true if it gets inside of the try or false if it gets inside of the catch statement.

Reply:
I have removed this unnecessary statement